examples/launch-tee.c

```diff
@@ -74,15 +74,6 @@ int main(int argc, char *const argv[])
         return -1;
     }
 
-    // Use the third (optional) command line argument as the disk image containing a data disk.
-    if (argc > 3) {
-        if (err = krun_set_data_disk(ctx_id, argv[3])) {
```

Comment:
Given that this is just an example, and probably we'll switch to double partitions quite soon, I think it'd be better to not change `launch_tee` and just specify `snp-example-data-disk.img` as `DATA_DISK_IMAGE`. That should work, shouldn't it?

Reply:
Sure. I rebased the commit to just use `snp-example-data-disk.img` as `DATA_DISK_IMAGE`. However, since we require that `DATA_DISK_IMAGE` to attest, I forced the usage of a `DATA_DISK_IMAGE` by failing if `argc != 4`.